crates/sui-core/src/execution_cache/object_locks.rs

```diff
@@ -156,14 +151,17 @@ impl ObjectLocks {
     fn clear_cached_locks(&self, locks: &[(ObjectRef, LockDetails)]) {
         for (obj_ref, lock) in locks {
             let entry = self.locked_transactions.entry(*obj_ref);
-            let occupied = match entry {
+            let mut occupied = match entry {
                 DashMapEntry::Vacant(_) => panic!("lock must exist"),
```

Comment:
Any reason we can't just not panic here as the fix?

Reply:
I will make it a debug panic only

Reply:
If we don't panic here, do we even need ref count?